File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int h;
    do{
    printf("Height: ");
     h=get_int();
    }while(h<0 | h > 23);
    int n=h+1;
    for(int i=0; i < h;i++){
        for(int j=0;j<n-(i+2);j++)
            printf(" ");
        for(int j=0;j<i+2;j++)
            printf("#");
        printf("\n");
    }
}

File: mario.py
import cs50

while True:
    print("Height:",end='')
    h  = cs50.get_int()
    if not (h<0 or h>23):
        break
n=h+1

for i in range(0,h):
    for j in range(0,n-(i+2)):
        print(" ",end = "")
    for j in range(0,i+2):
        print("#",end="")
    print ("")

